internal/service/cloudbackupsnapshotexportjob/resource_cloud_backup_snapshot_export_job.go

```diff
@@ -57,7 +57,7 @@ func returnCloudBackupSnapshotExportJobSchema() map[string]*schema.Schema {
 		},
 		"custom_data": {
 			Type:     schema.TypeSet,
-			Required: true,
+			Optional: true,
```

Comment:
I think this should be reflected in the changelog. WDYT?

Reply:
The docs already said it was `Optional`

Reply:
if doc said it was optional but it was required then looks like a bug that we are fixing?

Reply:
If the doc already mentioned this to be Optional (and this was always actually optional in the API) then it sounds like a bug fix, no? 
Otherwise, if this has been recently made Optional from API side, then we can use `note` in the changelog.

>IMO: Don't update the changelog, won't impact existing users unless they create a new resource.

Agree that it won't impact existing resources, but changelog isn't needed only for user impacting updates. It also facilitates transparency to users and to educate the ones who might have been working with previous incorrect behavior.

Reply:
Changed in 1b8fe28
Not 100% sure if it was optional on the API side so used a release `note`>
Thanks